rpcd/playbooks/roles/rpc_support/tasks/holland_config.yml

```diff
@@ -56,7 +56,7 @@
     minute: "{{ (ansible_eth0['ipv4']['address']|replace('.', '')|int)%60}}"
     hour: 23
     state: present
-    job: "/usr/local/bin/holland bk"
+    job: "{{ holland_venv_enabled | bool | ternary(holland_venv_bin + '/python2.7', '') }} {{ holland_venv_enabled | bool | ternary(holland_venv_bin + '/', '') }}holland bk"
```

Comment:
Kind of wondering if we should add /usr/local/bin to the right hand operator of the second ternary. Maybe it would be good to keep that in there.